apps/export/tasks/tasks_entries.py

```diff
@@ -29,9 +29,12 @@ def export_entries(export):
     leads_qs = Lead.objects.filter(project=export.project)
     if PP.Permission.VIEW_ALL_LEAD not in user_project_permissions:
         leads_qs = leads_qs.filter(confidentiality=Lead.Confidentiality.UNPROTECTED)
-    leads_qs = get_filtered_leads(filters, project, leads_qs)
+    # Lead and Entry FilterSet needs request to work with active_project
+    dummy_request = LeadGQFilterSet.get_dummy_request(project)
```

Comment:
Why are we creating a dummy request here?

Reply:
Both filters need request to generated dynamic filters so creating dummy request when calling from celery as there is no request to pass.